images/sds-lvm-csi/driver/controller.go

```diff
@@ -82,12 +84,16 @@ func (d *Driver) CreateVolume(ctx context.Context, request *csi.CreateVolumeRequ
 
 	var preferredNode string
 	if LvmBindingMode == BindingModeI {
```

Comment:
I would recommend to use switch in there (with default case) as we have only two kinds of possible operations.